metadata-ingestion/integrations.md

```diff
@@ -0,0 +1,7 @@
+import DataProviderComponent from '@site/src/pages/integrations'
```

Comment:
can we hide the right sidebar on this page? should be a docusaurus front matter config